packages/mdc-dialog/README.md

```diff
@@ -228,45 +213,77 @@ do so. We provide instructions on how to add ripples to buttons within the [mdc-
 | `deregisterSurfaceInteractionHandler(evt: string, handler: EventListener) => void` | Deregisters an event handler from the dialog surface element. |
 | `registerDocumentKeydownHandler(handler: EventListener) => void` | Registers an event handler on the `document` object for a `keydown` event. |
 | `deregisterDocumentKeydownHandler(handler: EventListener) => void` | Deregisters an event handler on the `document` object for a `keydown` event. |
-| `registerFocusTrappingHandler(handler: EventListener) => void` | Registers a focus event listener with a given handler at the capture phase on the document. |
-| `deregisterFocusTrappingHandler(handler: EventListener) => void` | Deregisters a focus event listener from a given handler at the capture phase on the document. |
-| `numFocusableTargets() => number` | Returns the number of focusable elements in the dialog |
-| `setDialogFocusFirstTarget() => void` | Resets focus to the first focusable element in the dialog |
-| `setInitialFocus() => void` | Sets focus on the `mdc-dialog__footer__button--accept` element. |
-| `getFocusableElements() => Array<Element>` | Returns the list of focusable elements inside the dialog. |
-| `saveElementTabState(el: Element) => void` | Saves the current tab index for the element in a way which it can be retrieved later on. |
-| `restoreElementTabState(el: Element) => void` | Restores the saved tab index (if any) for an element. |
-| `makeElementUntabbable(el: Element) => void` | Makes an element untabbable, e.g. by setting the `tabindex` to `-1`. |
-| `setBodyAttr(attr: string, val: string) => void` | Sets the given attribute to the given value on the body element. |
-| `rmBodyAttr(attr: string) => void` | Removes the given attribute from the body element. |
-| `getFocusedTarget() => Element` | Returns the currently focused element, e.g. `document.activeElement`. |
-| `setFocusedTarget(target: EventTarget) => void` | Sets focus on the given target, e.g. by calling `focus()` |
 | `notifyAccept() => {}` | Broadcasts an event denoting that the user has accepted the dialog. |
 | `notifyCancel() => {}` | Broadcasts an event denoting that the user has cancelled the dialog. |
+| `trapFocusOnSurface() => {}` | Sets up the DOM which the dialog is contained in such that focusability is restricted to the elements on the dialog surface (see [Handling Focus Trapping](#handling-focus-trapping) below for more details). |
+| `untrapFocusOnSurface() => {}` | Removes any affects of focus trapping on the dialog surface from the DOM (see [Handling Focus Trapping](#handling-focus-trapping) below for more details). |
+
+#### Handling Focus Trapping
 
+In order for dialogs to be fully accessible, they must conform to the guidelines outlined in
+https://www.w3.org/TR/wai-aria-practices/#dialog_modal. The main implication of these guidelines is
```

Comment:
"The main implication of these guidelines is that such that the only focusable elements..."

_Should read:_

"The main implication of these guidelines is that the only focusable elements..."